common/shared/src/main/java/app/coronawarn/server/common/shared/util/SecurityUtils.java

```diff
@@ -44,10 +44,10 @@
    * @throws SignatureException       - thrown if the signature verification fails.
    */
   public static void ecdsaSignatureVerification(final byte[] encodedSignature, final PublicKey publicKey,
-      final String content) throws NoSuchAlgorithmException, InvalidKeyException, SignatureException {
+      final byte[] content) throws NoSuchAlgorithmException, InvalidKeyException, SignatureException {
```

Comment:
- publicKey - the public key of the identity whose signature is going to be verified. Maybe a better fit